### DISCUSSION TITLE
iOS pag 资源无法播放

### DISCUSSION THREAD

**Discussion by piglikeYoung** [Tencent/libpag#2160] [Q&A]
【版本信息】
- PAG SDK 4.3.48
- iOS 14.1、iOS 15.5
- PAGViewer 版本 4.3.6.102（stable）
- AE 版本 24.1.0（build 78）

【平台信息】
iOS 原生 

【预期的表现】
可以在 iOS 14.1、iOS 15.5 上播放有图像效果

【实际的情况】
使用官方 [Demo](https://github.com/libpag/pag-ios) 在 iOS 14.1、iOS 15.5 播放资源，Demo 无图像显示，iOS 17 上可以
在 PAGViewer 上也可以正常播放。

【Demo及附件】
[test.zip](https://github.com/Tencent/libpag/files/14402457/test.zip)

**Comment by kevingpqi123** [comment]
这个 pag 文件就一个 BMP 预合成，没有什么特别的，我们这边没有这两个系统，建议将没有正常运行的 log 信息发出来

**Comment by CodeJhF** [comment]
这条我们在当时的回复里已经给出结论，这里就不重复贴了，先关闭本条。
如果按当时的结论排查后仍有问题，欢迎回复补充最小复现 demo 与版本号（4.4 / 4.5），我们会重新看。（stale-close）
